pom.xml

```diff
@@ -69,11 +69,46 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
-
+        <dependency>
```

Comment:
Why do you need this dependency? Remove if not needed.

Reply:
Trainer add this dependency